<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aeroForm</class>
 <widget class="QMainWindow" name="aeroForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>209</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AeroUber - a flight scheduling software</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start scheduling flights from requests in request file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Start scheduling</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>110</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>browseButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathSection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Open the request file:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
